src/application.coffee

```diff
@@ -77,7 +77,7 @@ logTypes =
 		humanName: 'Restarting application'
 
 logSystemMessage = (message, obj, eventName) ->
-	logger.log({ message, isSystem: true })
+	logger.log({ m: message, s: true })
```

Comment:
just noticed this, it'd be nice to use `s: 1` to save 3 bytes per message easily - I'm not sure the ui would even need any changes, @dfunckt @emirotin ?